Marquee/Cells/Table Cell/TableViewCell.swift

```diff
@@ -83,14 +83,14 @@ class TableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionView
         case 1:
             if let posterPath = tvShows?[indexPath.row].poster_path {
                 let url = URL(string: "https://image.tmdb.org/t/p/w500/\(posterPath)") ?? URL(string: "https://image.tmdb.org/t/p/w500/jrgifaYeUtTnaH7NF5Drkgjg2MB.jpg")
-                cell.poster.kf.setImage(with: url)}
+                let processor = RoundCornerImageProcessor(cornerRadius: 20)
```

Comment:
Interesting, is this an object that's given to you by KF?

Reply:
Yeah! It's a super helpful tool!